tests/componenttests/server/common/MessageBuilders.cpp

```diff
@@ -17,15 +17,17 @@
  * limitations under the License.
  */
 
-#include "MessageBuilders.h"
-#include "Constants.h"
 #include <string>
 
+#include "Constants.h"
+#include "MessageBuilders.h"
+#include "RialtoLogging.h"
+
 namespace firebolt::rialto::server::ct
 {
 ::rialto::SetConfigurationRequest createGenericSetConfigurationReq()
 {
-    constexpr int kLogLevel{63};
+    constexpr int kLogLevel{RIALTO_DEBUG_LEVEL_DEFAULT};
```

Comment:
why? debug prints should be present in component tests

Reply:
optionally, for better readability, you can define constexpr variable in Constants.h:
kAllLogs = RIALTO_DEBUG_LEVEL_FATAL | RIALTO_DEBUG_LEVEL_ERROR | RIALTO_DEBUG_LEVEL_WARNING | RIALTO_DEBUG_LEVEL_MILESTONE | RIALTO_DEBUG_LEVEL_INFO | RIALTO_DEBUG_LEVEL_DEBUG and assign it here (instead of using 63)

Reply:
> optionally, for better readability, you can define constexpr variable in Constants.h: kAllLogs = RIALTO_DEBUG_LEVEL_FATAL | RIALTO_DEBUG_LEVEL_ERROR | RIALTO_DEBUG_LEVEL_WARNING | RIALTO_DEBUG_LEVEL_MILESTONE | RIALTO_DEBUG_LEVEL_INFO | RIALTO_DEBUG_LEVEL_DEBUG and assign it here (instead of using 63)

I'll be honest that I just assumed the default log level would be the same as 63 but it isn't. I have implemented this suggestion.